_sass/_basic.scss

```diff
@@ -115,3 +115,35 @@ th, td {
 tr:nth-child(even) {
     background-color: $teachla-tint;
 }
+
+h4{
+    font-size: 135%;
+}
```

Comment:
Could we move this to a class instead (e.g. `member-name` or something)? It's not good practice to change element definitions unless we've got a great reason to do so!